presto-druid/src/main/java/io/prestosql/plugin/druid/DruidJdbcClient.java

```diff
@@ -267,4 +294,76 @@ public void dropSchema(JdbcIdentity identity, String schemaName)
     {
         throw new PrestoException(DruidErrorCode.DRUID_DDL_NOT_SUPPORTED, "DDL operations are not supported in the presto-druid connector");
     }
+
+    private Set<AggregateFunctionRule> aggregateFunctionRules()
+    {
+        JdbcTypeHandle bigIntHandle = new JdbcTypeHandle(Types.BIGINT, Optional.of("bigint"), 0, 0, Optional.empty(), Optional.empty());
+        JdbcTypeHandle doubleHandle = new JdbcTypeHandle(Types.DOUBLE, Optional.of("double"), 0, 0, Optional.empty(), Optional.empty());
+        Set<Type> druidNumericTypes = Set.of(DOUBLE, BIGINT, REAL);
+        Set<Type> numericAndTimeStampType = Set.of(DOUBLE, BIGINT, REAL, TimestampType.createTimestampType(3));
+        return ImmutableSet.<AggregateFunctionRule>builder()
+                .add(new ImplementCountAll(bigIntHandle))
+                .add(new ImplementCount(bigIntHandle))
+                .add(SingleInputAggregateFunction.builder()
+                        .prestoName("approx_distinct")
+                        .druidName("approx_count_distinct")
+                        .jdbcTypeHandle(bigIntHandle)
+                        .outputType(BIGINT)
+                        .build())
+                .add(SingleInputAggregateFunction.builder()
+                        .prestoName("max")
+                        .inputTypes(numericAndTimeStampType)
```

Comment:
Maybe we can make the match based on function signature.
Eg `max(double)`, `max(bigint)`, `max(real)`.
This would increase verbosity (`numericAndTimeStampType` is more concise),
but hopefully would increase overall readability as well.